diff --git a/ccx_2.17/src/ccx_2.17.c b/ccx_2.17/src/ccx_2.17.c
index 7933c90..d1e6aa1 100644
--- a/ccx_2.17/src/ccx_2.17.c
+++ b/ccx_2.17/src/ccx_2.17.c
@@ -15,9 +15,9 @@
 /*     along with this program; if not, write to the Free Software       */
 /*     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.         */
 
-#ifdef __WIN32
-_set_output_format(_TWO_DIGIT_EXPONENT);
-#endif
+// #ifdef __WIN32
+// _set_output_format(_TWO_DIGIT_EXPONENT);
+// #endif
 
 #ifdef CALCULIX_MPI
 #include <spoolesMPI.h>
@@ -38,6 +38,9 @@ struct timespec totalCalculixTimeStart,totalCalculixTimeEnd;
 
 int main(int argc,char *argv[])
 {
+  #ifdef __WIN32
+  _set_output_format(_TWO_DIGIT_EXPONENT);
+  #endif
   
   FILE *f1;
     
