diff --git a/ccx_2.23/src/orifice.f b/ccx_2.23/src/orifice.f
index 7fb9769..13f164f 100644
--- a/ccx_2.23/src/orifice.f
+++ b/ccx_2.23/src/orifice.f
@@ -34,7 +34,7 @@
       integer nelem,nactdog(0:3,*),node1,node2,nodem,numf,
      &     ielprop(*),nodef(*),idirf(*),index,kflag,
      &     inv,ipkon(*),kon(*),number,kgas,nelemswirl,
-     &     nodea,nodeb,iaxial,mi(*),i,itype,iplausi
+     &     nodea,nodeb,iaxial,mi(*),i,itype,iplausi,curve
 !     
       real*8 prop(*),v(0:mi(2),*),xflow,f,df(*),kappa,R,a,d,dl,
      &     p1,p2,T1,Aeff,C1,C2,C3,cd,cp,physcon(*),p2p1,km1,dvi,
@@ -42,7 +42,7 @@
      &     rad,beta,reynolds,theta,k_phi,c2u_new,u,pi,xflow_oil,
      &     ps1pt1,uref,cd_chamf,angle,vid,cdcrit,T2,radius,
      &     initial_radius,co(3,*),vold(0:mi(2),*),offset,ttime,time,
-     &     x_tab(100), y_tab(100),x_tab2(100),y_tab2(100),curve,xmach
+     &     x_tab(100), y_tab(100),x_tab2(100),y_tab2(100),xmach
 !
       pi=4.d0*datan(1.d0)   
       if(kflag.eq.0) then
