diff --git a/ccx_2.23/src/initialconditionss.f b/ccx_2.23/src/initialconditionss.f
index 328a740..a58303a 100644
--- a/ccx_2.23/src/initialconditionss.f
+++ b/ccx_2.23/src/initialconditionss.f
@@ -989,12 +989,13 @@ c     endif
      &                 ipol,inl,ipoinp,inp,ipoinpc)
                   if((istat.lt.0).or.(key.eq.1)) return
                   do j=1,jmax
-                    read(textpart(j+2)(1:20),'(f20.0)',
+                    read(textpart(j)(1:20),'(f20.0)',
      &                   iostat=istat) beta(j)
-                    if(istat.gt.0) 
-     &                   call inputerror(inpc,ipoinpc,iline,
-     &                   "*INITIAL CONDITIONS%",ier)
-                    return
+                    if(istat.gt.0) then
+                      call inputerror(inpc,ipoinpc,iline,
+     &                     "*INITIAL CONDITIONS%",ier)
+                      return
+                    endif
                     xstate(ntot+j,k,l)=beta(j)
                   enddo
                   ntot=ntot+jmax
