diff --git a/ccx_2.23/src/e_c3d_rhs_th.f b/ccx_2.23/src/e_c3d_rhs_th.f
index 92af34b..62b872f 100644
--- a/ccx_2.23/src/e_c3d_rhs_th.f
+++ b/ccx_2.23/src/e_c3d_rhs_th.f
@@ -291,7 +291,8 @@ c     if(nload.gt.0) then
                   call dflux(xload(1,id),t1l,istep,iinc,tvar,
      &                 nelem,kk,pgauss,jltyp,temp,press,sideload(id),
      &                 areaj,vold,co,lakonl,konl,ipompc,nodempc,coefmpc,
-     &                 nmpc,ikmpc,ilmpc,iscale,mi)
+     &                 nmpc,ikmpc,ilmpc,iscale,mi,
+     &                 sti,xstateini,xstate,nstate_,dtime)
                   if((nmethod.eq.1).and.(iscale.ne.0))
      &                 xload(1,id)=xloadold(1,id)+
      &                 (xload(1,id)-xloadold(1,id))*reltime
@@ -432,7 +433,8 @@ c     endif
                   call dflux(xload(1,id),temp,istep,iinc,tvar,
      &                 nelem,i,coords,jltyp,temp,press,sideload(id),
      &                 areaj,vold,co,lakonl,konl,ipompc,nodempc,
-     &                 coefmpc,nmpc,ikmpc,ilmpc,iscale,mi)
+     &                 coefmpc,nmpc,ikmpc,ilmpc,iscale,mi,
+     &                 sti,xstateini,xstate,nstate_,dtime)
                   if((nmethod.eq.1).and.(iscale.ne.0))
      &                 xload(1,id)=xloadold(1,id)+
      &                 (xload(1,id)-xloadold(1,id))*reltime
